docs/next/styles/globals.css

```diff
@@ -104,6 +104,10 @@ dt > a.reference.internal {
   @apply inline-flex items-center px-3 py-0.5 rounded-full align-middle text-xs uppercase font-medium bg-sea-foam text-gable-green
 }
 
+.preview-tag {
```

Comment:
Do we also need a `.preview-tag.flag.preview` for the API docs? Looking at superseded and deprecated, we have two different CSS per flag, but it's not the case for experimental.

Reply:
That CSS hasn't been touched in quite some time, but it looks like @hellendag was the one to do it. Perhaps he would know. 🤔

Reply:
I think the `[foo]-tag` classname is added in MDX components, and `flag [foo]` is added via Sphinx, and I think to be safest we would want to include both selectors.

```
.preview-tag,
.flag.preview {
  ...
}
```

So any of the rules where they are currently separate (e.g. `.superseded-tag` has a separate rule from `.flag.superseded`), we probably want them to be combined instead.

Additionally:

- There are two rules below that have `.legacy-tag`, but only one should. (I think probably not the `flag.superseded` one?)
- `.deprecated-tag` and `.flag.deprecated` have different styles, which seems like it probably wasn't intentional. Maybe these should be unified?

Reply:
Thanks @hellendag ! I updated the CSS for the preview flag [c70913b](https://github.com/dagster-io/dagster/pull/26819/commits/c70913b135f0ce1376ce168f63e9fca50aa2838d) and fixed the superseded and deprecated ones [here](https://github.com/dagster-io/dagster/pull/26972).

Reply:
CSS here and in the other PR lgtm, thanks!